rskj-core/src/main/java/co/rsk/peg/federation/StandardMultisigFederation.java

```diff
@@ -34,13 +35,14 @@
 
 public class StandardMultisigFederation extends Federation {
 
-    public StandardMultisigFederation(
+    protected StandardMultisigFederation(
```

Comment:
Test classes should also be moved into a `federation` package in `tests` package